polkadot-protocol/protocol-components/accounts/account-data-structure.md

```diff
@@ -0,0 +1,132 @@
+---
+title: Account Data Structure
+description: TODO
+---
+
+# Account Data Structure
+
+!!! warning "Update Notice"
+    Please read Substrate to [Polkadot SDK](https://docs.substrate.io/polkadot-sdk/) page first.
+
+Accounts are a foundational element of any blockchain, and in Substrate, they serve various functions. This article explains how accounts are stored in Substrate and how their data structures are utilized to manage the account lifecycle within the runtime logic.
+
+## Account
+
+The `Account` data type is a generically defined storage map within the [`frame-system`](https://paritytech.github.io/polkadot-sdk/master/src/frame_system/lib.rs.html) pallet:
+
+```rs
+/// The full account information for a particular account ID.
+#[pallet::storage]
+#[pallet::getter(fn account)]
+pub type Account<T: Config> = StorageMap<
+  _,
+  Blake2_128Concat,
+  T::AccountId,
+  AccountInfo<T::Nonce, T::AccountData>,
+  ValueQuery,
+>;
+```
+
+The StorageMap for an Account consists of the following parameters:
+
+- The first parameter `(_)` is used in macro expansion.
+- `Blake2_128Concat` specifies the hashing algorithm to use.
+- `T::AccountId` is used as the key for over the `AccountInfo<T::Nonce, T::AccountData>` struct.
+
+!!! note
+    See [StorageMap API](https://paritytech.github.io/polkadot-sdk/master/frame_support/storage/types/struct.StorageMap.html) for more details.
+
+## Account Info
+
+The `AccountInfo` for an account is also defined within the [`frame-system`](https://paritytech.github.io/polkadot-sdk/master/src/frame_system/lib.rs.html) pallet:
+
+```rs
```

Comment:
If we do choose to leave code snippets, then we should put them in snippets, but I am more leaning towards just linking the line in the code base.